apps/mobile/src/components/ChatDetailScreen/components/UserAvatarDetail.tsx

```diff
@@ -0,0 +1,66 @@
+import {BackHandler, Modal, useWindowDimensions} from 'react-native'
+import {userAvatarDetailAtom} from '../../../state/chat/atoms/showUserAvatarDetailAtom'
+import {useAtom} from 'jotai'
+import {Stack} from 'tamagui'
+import {useSafeAreaInsets} from 'react-native-safe-area-context'
+import IconButton from '../../IconButton'
+import closeSvg from '../../images/closeSvg'
+import {ImageZoom} from '@likashefqet/react-native-image-zoom'
+import {useFocusEffect} from '@react-navigation/native'
+import React from 'react'
+
+function UserAvatarDetail(): JSX.Element {
```

Comment:
Can you please rename this file so it is apparent this is overlay/modal. I suggest `ImageZoomOverlay`